maestro/audit.py

```diff
@@ -378,21 +378,16 @@ def _format_container_dict(self, what):
         for item in what:
             x = dict(ship=item.ship.ip, service=item.service.name)
             d[item.name] = x
-        d_str = '--hostserv={}'.format(StringIO.StringIO(json.dumps(d)).read())
+        d_str = '--hostserv={}'.format(StringIO(json.dumps(d)).read())
```

Comment:
The whole thing about `StringIO` was if you were going to use it to pass stuff via `stdin` to the script. If you're not, you don't need to use it. But I'd rather you pass stuff in via `stdin`, so as to not run into the command-line size problem I mentioned earlier.